tests/integration/client.rs

```diff
@@ -122,3 +122,21 @@ async fn test_iter() {
     }
     assert_eq!(i, 13);
 }
+
+#[async_std::test]
+async fn fetch_system_chain() {
+    let node_process = test_node_process().await;
+    node_process.client().rpc().system_chain().await.unwrap();
+}
```

Comment:
Actually could you combine these into one single test? Spinning up 3 separate nodes for this will unnecessarily increase CI times.

Reply:
Good point. done